src/python/include/Simplex_tree_interface.h

```diff
@@ -188,6 +189,16 @@ class Simplex_tree_interface : public Simplex_tree<SimplexTreeOptions> {
     // this specific case works because the range is just a pair of iterators - won't work if range was a vector
     return Base::skeleton_simplex_range(dimension).end();
   }
+
+  Boundary_simplex_iterator get_boundary_iterator_begin(const Simplex& simplex) {
+    // this specific case works because the range is just a pair of iterators - won't work if range was a vector
+    return Base::boundary_simplex_range(Base::find(simplex)).begin();
+  }
+
+  Boundary_simplex_iterator get_boundary_iterator_end(const Simplex& simplex) {
+    // this specific case works because the range is just a pair of iterators - won't work if range was a vector
+    return Base::boundary_simplex_range(Base::find(simplex)).end();
```

Comment:
Calling Base::find twice (for begin and end) is not nice.

Reply:
Do you want me to save the `Simplex_handle` returned by `Base::find(simplex)` from the `get_boundary_iterator_begin` call and save it in a class attribute ? `get_boundary_iterator_end` could be without arguments.

Reply:
The cleanest would be to also call boundary_simplex_range only once. But if that's complicated, storing the Simplex_handle would be fine, although "class attribute" seems strange (what class?). Couldn't the C++ interface return a pair of iterators (I think cython understands std::pair)?

Reply:
I return a pair of iterators on 458bc2d